api/src/main/java/org/openmrs/util/OpenmrsClassLoader.java

```diff
@@ -590,7 +590,6 @@ protected static void nullInstance(Object instance) {
 				field.setAccessible(true);
 				if (Modifier.isStatic(mods) && Modifier.isFinal(mods)) {
 					// Doing something recursively is too risky
-					continue;
```

Comment:
Can't we simplify this by only having an if statement without else?

Reply:
Fixed 😊